### ISSUE TITLE
is there E2E test for dynamic shape?

### ISSUE THREAD

**Issue by FullZing** [onnx/onnx-mlir#1913]
or say,  what's the role of onnx-mlir in the case of dynamic shape?

**Comment by chentong319** [comment]
Yes. We modify the backend tests provided by onnx package to test dynamic shape. The target name 'check-onnx-backend-dynamic'
